airflow/providers/docker/hooks/docker.py

```diff
@@ -87,6 +87,7 @@ def construct_tls_config(
         ca_cert: str | None = None,
         client_cert: str | None = None,
         client_key: str | None = None,
+        verify: bool | None = None,
```

Comment:
Even though the `TLSConfig` init accepts `None`, and since it considers it `False`, it's better to only accept bool in our class and set default to `False`:
```suggestion
        verify: bool = False,
```